core/runtime/src/main/java/io/quarkiverse/langchain4j/guardrails/AbstractJsonExtractorOutputGuardrail.java

```diff
@@ -0,0 +1,38 @@
+package io.quarkiverse.langchain4j.guardrails;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import dev.langchain4j.data.message.AiMessage;
+import jakarta.inject.Inject;
+import org.jboss.logging.Logger;
+
+import static io.quarkiverse.langchain4j.guardrails.GuardrailsUtils.trimNonJson;
+import static io.quarkiverse.langchain4j.guardrails.GuardrailsUtils.validateJson;
+
+public abstract class AbstractJsonExtractorOutputGuardrail implements OutputGuardrail {
+
+    private static final ObjectMapper MAPPER = new ObjectMapper();
```

Comment:
I'm not sure we want to do that.
Mapper are huge, you probably want to reuse an existing one, maybe even the Quarkus one.

Reply:
`@Inject ObjectMapper MAPPER`

Reply:
Yes, that would be better (and it will also handle the modules the user has registered)

Reply:
Sorry, this is a leftover of my original implementation, this mapper is totally useless now and I will remove it.